lib/setup.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+// configures a hapi server that can be used
+// for unit testing outside of docker
+
+const hapi = require('hapi');
+const ObjectID = require('mongodb').ObjectID;
+const mongo = require('mockgo');
+const get = require('../methods/get.js').method;
+const track = require('../methods/track.js').method;
+const extractInfo = require('../methods/extractInfo.js').method;
+const types = require('../methods/types.js').method;
+
+module.exports = (options, callback) => {
+  const server = new hapi.Server({});
```

Comment:
thats fine for now.  We'll want to change at some point because hapi isn't technically a dependency